yellowbrick/regressor/base.py

```diff
@@ -60,7 +62,5 @@ def score(self, X, y, **kwargs):
         score : float
             The R^2 score of the underlying regressor
         """
-        raise NotImplementedError(
-            "Subclasses of RegressionScoreVisualizer must implement score "
-            " and return an R^2 score of the underlying estimator"
-        )
+        self.score_ = self.estimator.score(X, y)
+        return self.score_
```

Comment:
Oh! This is so much better now that we have a more comprehensive sense of what the regressors module is like (would have been much harder to anticipate this three years ago 😊)!